meta/labs_program/meetings/15-08-17_selecting_initial_labs_team.md

```diff
@@ -0,0 +1,176 @@
+## Actually Selecting Initial Labs Team
+
+_In attendance:_
+
+- Dave Fontenot
+- Jonathan Leung
+- Zach Latta
+
+-------------------------------------------------------------------------------
+
+## hackEDU: Identifying women leaders in high school tech
+
+- CodeCamp
+- Girls Who Code
+- NCWIT
+- Ask freshman women for awesome high schoolers they are mentoring
+  - mentorship between college <> high school
+- She++
+- Graph Search: my female friends in HS Hackers
+
+Clubs are led by more than one person
+
+- convince male leaders of the importance of bringing on a female co-leader
+
+Personally mentoring to sure up technical strength?
+
+## hackEDU Advantages: what do we offer that HS students want?
+
+Leadership Titles
+
+Diverse co-organizers in the past have had distinct advantages
+
+— x, y, z
+
+## Early Metrics
```

Comment:
Was there something that was supposed to be under this heading, because it is currently a blank line before another heading